### DISCUSSION TITLE
Release v1.5.0.0

### DISCUSSION THREAD

**Discussion by randypacker** [dreamworksanimation/openmoonray#138] [Announcements]
### Build Improvements
* Upgraded to Embree 4.2.0
* Upgraded to Optix 7.6
* Added lsb to the container build scripts 
* Added support for VFX Reference Platform 2023, includes implicit support for Rocky9, GCC-11 and Python-3.10
### Features
* Added initial implementation of a new adaptive light sampling scheme
* Implemented an improved dw_acos() over acos() to the scene_rdl2 math lib
* Added automatic CPU-affinity control to MoonRay, with own thread pool
* Exposed the execution mode (auto, xpu, vector, scalar) in hdMoonray
* Added a few Geo preview settings to hdMoonray
* Add tolerance for float equals and new vector equals mode to the OpMap
* Several improvements to DwaToonMaterial for non-photorealistic looks
### Bug Fixes
* Removed a duplicate CPU handler in XPU mode
* Fixed XPU debug asserts
* Fixed XPU fallback behavior to only fallback when run in auto mode
* Fixed ShadowReceiverSets in XPU mode
* Fixed exclusion symbol ^ in LPEs.
* Fixed local motion blur to use the region with maximum reduction when within overlapping regions
* Fixed VDB light filter density_grid_name attribute type
* Fixed allocation of OIIO texturing time in the log output 
### Other
* Renamed debug and info variables
* Removed variance AOVs as part of a refactor
* Removed Dual code as part of a refactor
* Removed unused ray sorting code in XPU mode
* Removed and archived the PathIntegratorOneSampler
* Remove unused and unsupported multi-film support in the AOV system
* Removed deprecated attributes from SceneVariables
* Added Displacement map parameters to the documentation
* Added stats for bundled/gpu intersection rays
* Extended XPU queue support for regular rays
* Combined primary and incoherent ray queues
* Miscellaneous rdl2_print improvements and refactor
* With vfxrefplat-2023, hdMoonray can be running in Nuke 15, initial check shows it running.

<hr /><em>This discussion was created from the release <a href='https://github.com/dreamworksanimation/openmoonray/releases/tag/openmoonray-1.5.0.0'>Release v1.5.0.0</a>.</em>

**Comment by mpjohn** [comment]
Exciting to see a new release. Are there any instructions on how to build this against a specific version of Houdini? Or lacking that - does the new python update suggest a version of Houdini that the Rocky 9 build will work with ( without a rebuild )?

Thanks

**Comment by giaginsk** [comment]
I would like to understand how to install it
